### PR TITLE
Implement java.lang.Character.charCount

### PR DESCRIPTION
**Pull request by kxbmap**

### CODE REVIEW

**Comment by gzm0** [comment]
Jenkins, ok to test

File: test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala [hunk 1]
```diff
@@ -47,6 +47,25 @@ class CharacterTest {
     assertEquals('9', Character.forDigit(9, 10))
   }
 
+  @Test def charCount(): Unit = {
+    def random(from: Int, to: Int): Int =
+      scala.util.Random.nextInt(to - from + 1) + from
```
Comment (gzm0) [inline test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala]: It is OK to make random tests, but please seed the random generator to get reproducible results.

Comment (kxbmap) [inline test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala]: OK, I'll fix it.

**Comment by sjrd** [comment]
LGTM Thanks!

**Pull request merged**